elFinder.NetCore/Exceptions/FileTypeNotAllowedException.cs

```diff
@@ -18,10 +18,5 @@ public FileTypeNotAllowedException(string message, Exception innerException)
             : base(message, innerException)
         {
         }
-
-        protected FileTypeNotAllowedException(SerializationInfo info, StreamingContext context)
```

Comment:
@utilsites What's the reasoning for removing this constructor?

Reply:
Because it causes the following warning when rebuilding project:
\elFinder.NetCore\elFinder.NetCore\Exceptions\FileTypeNotAllowedException.cs(23,9,23,30): warning SYSLIB0051: 'ApplicationException.ApplicationException(SerializationInfo, StreamingContext)' is obsolete: 'This API supports obsolete formatter-based serialization. It should not be called or extended by application code.' (https://aka.ms/dotnet-warnings/SYSLIB0051)